replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
+---
+
+(1) Replicated table:
+
+```{r echo = FALSE}
```

Comment:
I understand echo = false, because it was probably hiding the code and producing the stargazer table, right?  I used it like that to hide the gears that were churning out figures/summaries that would still show when knitted up.